### PR TITLE
MCS-2981 modify the webhook tutorial for export function.

### PR DESCRIPTION
**Pull request by dariachen**
Hi @anchi1216  and @kevingo 
Please help to review. Thanks!

### CODE REVIEW

File: content/zh-CN/tutorial/getting_started.md [hunk 1]
```diff
@@ -99,6 +99,8 @@
 
 步骤三，储存此 JSON 档案。
 
+请注意，在汇出的 JSON 档案中，我们会以 http://localhost 取代您在**触发条件与动作**中设定的Webohook URL，以防止他人在直接汇入此J SON 档案后，将其装置资讯上传至您的服务器。
+
```
Comment (anchi1216) [inline content/zh-CN/tutorial/getting_started.md:103]: J SON 好像多了一個空格喔~

File: content/zh-TW/tutorial/getting_started.md [hunk 2]
```diff
@@ -77,6 +77,8 @@
 
 步驟三，儲存此 JSON 檔案。
 
+請注意，在匯出的 JSON 檔案中，我們會以 http://localhost 取代您在**觸發條件與動作**中設定的 Webohook URL，以防止他人在直接匯入此 JSON 檔案後，將其裝置資訊上傳至您的伺服器。
```
Comment (kevingo) [inline content/zh-TW/tutorial/getting_started.md]: webhook

File: content/zh-CN/tutorial/getting_started.md [hunk 3]
```diff
@@ -99,6 +99,8 @@
 
 步骤三，储存此 JSON 档案。
 
+请注意，在汇出的 JSON 档案中，我们会以 http://localhost 取代您在**触发条件与动作**中设定的Webohook URL，以防止他人在直接汇入此J SON 档案后，将其装置资讯上传至您的服务器。
```
Comment (kevingo) [inline content/zh-CN/tutorial/getting_started.md]: webhook

**Comment by dariachen** [comment]
Thanks @anchi1216 , 
Fixed that!

**Comment by dariachen** [comment]
Thanks @kevingo !
I fixed the typos.

**Comment by anchi1216** [comment]
looks good =D

**Pull request merged**